manifest/readme.md

```diff
@@ -0,0 +1,57 @@
+### Performance Analyzer
+
+#### Overview
+
+The [Performance Analyzer NuGet](https://www.nuget.org/packages/Skyline.DataMiner.Utils.PerformanceAnalyzer) allows you to gather the performance metrics for a solution you have implemented, but collecting the data is only one piece of the puzzle. For the collected metrics to bring value, you need to be able to interpret them and use them easily for decision-making. As humans, we often struggle to comprehend numbers, particularly when they are extremely large or, as is often the case with performance metrics, very small. This makes it essential to provide a more **intuitive way to interpret the data**. This Performance Analyzer package helps you with this by providing a **low-code app that visualizes the metrics out of the box** along with a **collection of GQI queries that allow for custom visualizations**, tailored to the specific requirements of a project.
+
+![Performance Analyzer LCA](./Images/lca_timeline.png)
+
+> [!IMPORTANT]
+> This package supplements the [Performance Analyzer NuGet](https://www.nuget.org/packages/Skyline.DataMiner.Utils.PerformanceAnalyzer). Implementing it in your code is a prerequisite.
+
+> [!NOTE] 
+> For a detailed introduction to the Performance Analyzer NuGet and to DataMiner Low-Code Apps, refer to DataMiner Docs:
+>
+> - [Performance Analyzer](https://docs.dataminer.services/user-guide/Standard_Apps/Monitoring_Solutions/Performance_Analyzer/Performance_Analyzer.html)
+> - [DataMiner Low-Code Apps](https://docs.dataminer.services/user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Low_Code_Apps/Application_framework.html)
```

Comment:
As long as the documentation hasn't been published yet on docs, it would be better to hide this note for now. It's better not to provide a link than to provide one that isn't working.